### PR TITLE
feat: Add OWNERS_ALIASES and edit OWNERS file

### PR DESCRIPTION
**Pull request by opokornyy**
**What this PR does / why we need it**:

This PR should automate  adding sradco to all monitoring PRs

**Release note**:
```
None
```

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,13 +1,28 @@
...
-  - 0xFelix
-  - akrejcir
-  - ksimon1
-  - lyarwood
-  - jcanocan
-  - opokornyy
-  - codingben
+# See the OWNERS docs at https://go.k8s.io/owners
+
+filters:
```
Comment (0xFelix) [inline OWNERS:3]: You could add `docs/metrics.dm` and `‎tests/monitoring_test.go` too.

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+aliases:
+  approvers:
+    - 0xFelix
+    - akrejcir
+    - ksimon1
+    - lyarwood
+  code-reviewers:
```
Comment (0xFelix) [inline OWNERS_ALIASES:9]: Add newline between groups?

File: OWNERS_ALIASES [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    - lyarwood
+    - jcanocan
+    - opokornyy
+    - codingben
+
+  #
+  # SIG Monitoring
+  #
+  sig-monitoring:
+    - sradco
```
Comment (lyarwood) [inline OWNERS_ALIASES:23]: @sradco should others also be listed here?

**Review by lyarwood — COMMENTED** [review]
This `/lgtm` but I wonder if there's a way of testing this somewhere before we merge?

File: OWNERS_ALIASES (hunk 3 above)
Comment (opokornyy) [reply OWNERS_ALIASES:23]: I have asked @sradco yesterday and there is no need to add someone else now.

**Review by 0xFelix — APPROVED** [review]
Thanks!

/approve

**Pull request merged**